Resources/Prototypes/_CD/Traits/traits.yml

```diff
@@ -0,0 +1,12 @@
+﻿- type: trait
+  category: Quirks
+  id: Synthetic
+  name: trait-synth-name
+  description: trait-synth-desc
+  blacklist:
+    components:
+      - Silicon # The only component I could find that is exclusive to IPCs
+      - BorgChassis
```

Comment:
Is the blacklist not redundant with excludedSpecies?

Reply:
change to `# DeltaV - blacklist IPCs`